docs/pools.adoc

```diff
@@ -10,43 +10,27 @@ Enable and start the `kadalu-mgr`
 [root@server1 ~]# systemctl start kadalu-mgr
 ----
 
-[source,console]
-----
-[root@server1 ~]# kadalu pool create DEV
-Pool created successfully.
-ID: 4e67989e-0c24-44ac-b0a7-81aa5cb4ec6b
-----
-
-Run `pool list` command to see the list of Pools
-
-[source,console]
-----
-[root@server1 ~]# kadalu pool list
-                                  ID  Name
-4e67989e-0c24-44ac-b0a7-81aa5cb4ec6b  DEV
-----
-
-The pool data and the configurations are stored in `/var/lib/kadalu`. Since the storage manager is running in a single node, by default it is not highly available. Use `kadalu-lite` Volume for high availability, or backup the Pool data in regular intervals. Refer "High availability" chapter (Coming soon) for more details.
+The cluster configurations are stored in `/var/lib/kadalu`. Since the storage manager is running in a single node, by default it is not highly available. Use `kadalu-lite` Volume for high availability, or backup the Pool data in regular intervals. Refer "High availability" chapter (Coming soon) for more details.
 
-Since Kadalu Storage manager supports multiple Pools, the manager node is not part of any Pool by default. Optionally invite the Storage manager node to join the Pool, which is similar to inviting other Storage nodes.
+No nodes are added to the cluster by default. Invite the node to become part of the Cluster by running the following command.
```

Comment:
Does manager node be part of of any pool by default.?
If manager node is invited be a part of particular pool, can it also manage other pools?

Reply:
nodes are not limited to a pool now. A node can be part of one or more storage pools